src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.csproj

```diff
@@ -11,11 +11,23 @@
     <PreserveCompilationContext>true</PreserveCompilationContext>
     <OutputType>exe</OutputType>
     <X86ProjectDirectory>..\..\tools\Microsoft.AspNetCore.Mvc.Razor.ViewCompilation-x86\</X86ProjectDirectory>
+
+    <!--
+      This prevents dotnet from automatically generating a lib/tfm/*.dll; in turn we're then able
+      to exclude the lib/net461 and manually construct only the necessary folders:
+      - lib/netcoreapp2.0/...
+      - runtimes/win7-x64/lib/...
+      - runtimes/win7-x86/lib/...
+    -->
+    <IncludeBuildOutput>false</IncludeBuildOutput>
   </PropertyGroup>
 
   <ItemGroup>
     <None Include="build\**\*" Pack="true" PackagePath="%(Identity)" />
-    <None Include="$(X86ProjectDirectory)\bin\$(Configuration)\net461\win7-x86\$(MSBuildThisFileName)-x86.exe" Pack="true" PackagePath="lib\net461\$(MSBuildThisFileName)-x86.exe" />
```

Comment:
Because of the funky "import" feature that NuGet and the SDK have added that munges net461/ns2.0/netcoreapp2.0 compatibility, it would be good to add a `lib/net461/_._` placeholder. It makes asset selection more predictable.